docs/dev/developer-guides/transactions/transactions.md

```diff
@@ -1,61 +1,87 @@
-- **Value**: The quantity of Wei or Ether that the sender account wishes to transmit to the recipient is represented by the value.
-- **Data**: If the transaction receiver is a smart contract, the data contains information for the contract's functions to be executed. This comprises data with varying lengths.
-- **Signature**: A signature indicates who sent the communication. The signature is created when an externally owned account confirms and signs the transaction with its private key.
+### Block explorer data
 
-### Transaction Types
+The following general information is included in the transaction details on the [block explorer](https://explorer.zksync.io/transactions/).
 
-- Simple or asset transfers: This refers to the regular tokens transfer in the form of ether from one account to another.
+- `Transaction Hash`: Unique 66 character identifier generated when transaction submitted to L2.  
+- `Commit Tx hash`: Unique 66 character identifier generated when transaction committed to L1.  
+- `Prove Tx hash`: Unique 66 character identifier generated when transaction proven on L1. 
+- `Execute Tx hash`: Unique 66 character identifier generated when transaction executed on L1. 
+- `Status`: One of `Pending`, `Included`, `Verified`, or `Failed`. See [Transaction statuses section](#transaction-statuses) below.
+- `Block`: Block number containing the transaction.  
+- `Batch`: Batch number containing the transaction. 
+- `From`: The account or smart contract address sending the transaction. 
+- `Tokens Transferred`: Details on all tokens transferred by the transaction. 
+- `Contract address`:  The transaction recipient. 
+- `Input data`: Any additional data used by the transaction for verified contracts.
+- `Fee`: Fee for the tx in ETH and USD value. Click **More Details** to see info on refunds.
+- `Nonce`: Sender nonce.                         
+- `Created`: Timestamp of when the transaction was added to the block.   
 
-- Contract deployment transactions: Contract deployment on zkSync is quite different from Ethereum.
-  - Ethereum: Contract deployment occurs when a user sends a transaction to the zero address `(0x000...000)` with the `data` field of the transaction equal to the contract bytecode concatenated with the constructor parameters.
-  - zkSync Era: To deploy a contract on zkSync, a user calls the `create` function of the [ContractDeployer](../system-contracts.md#contractdeployer) and provides the hash of the contract to be published, as well as the constructor arguments. The contract bytecode itself is supplied in the `factory_deps` field of the EIP712 transactions. If the contract is a factory (i.e. it can deploy other contracts), the contracts bytecodes should be included in the `factory_deps`.
-  Read more on [contract deployment](../../building-on-zksync/contracts/contract-development.md).
+## Contract deployment differences between zkSync Era and Ethereum
 
```

Comment:
```suggestion
Contract deployment is a special kind of transaction. A contract deployment creates a new contract account in the system. We list the differences for this kind of transaction between zkSync Era and Ethereum.
```
If we have a page that explains what type of accounts (specifically, contracts) there are, it's even better